Core/Data/LanguageId.cs

```diff
@@ -16,7 +16,7 @@ public enum LanguageId : uint8_t
 		chinese_traditional,
 		id_12,
 		portuguese,
-		blank = 254,
-		end = 255
+		//blank = 254,
```

Comment:
these were not the cause of the bug, but they also just weren't necessary to include here